packages/amazon-ssm-agent/amazon-ssm-agent.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=amazon-ssm-agent
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/amazon-ssm-agent
+KillMode=process
+
+# Restart the agent regardless of whether it crashes (and returns a non-zero result code) or if
+# is terminated normally (e.g. via 'kill -HUP').  Delay restart so that the agent is less likely
+# to restart during a reboot initiated by a script. If the agent exits with status 194 (reboot
+# requested), don't restart at all.
+Restart=always
+RestartPreventExitStatus=194
+RestartSec=90
```

Comment:
I think that we could get away with a lower value if this became a concern. I am absolutely fine with 90 seconds until we learn otherwise.